### DISCUSSION TITLE
Facial Recognition - Tweaking/Correcting

### DISCUSSION THREAD

**Discussion by finickyFalcon5** [immich-app/immich#14266] [Q&A]
My daughter and son looked very similar when they were very young. Immich detected my daughter but did not detect my son at all. All the images with my son are also detected as my daughter.
Is there a way to create a new person, and then manually mark some images as the new person, so that immich ML can learn to detect the new person?
Is there a way to remove some images from a person's profile?

**Comment by davey-g-uk** [comment]
To answer some parts of your question:

"Is there a way to create a new person, and then manually mark some images as the new person" - Yes, when you look at a photo where son is detected as your daughter, get the sidebar up with info and click the pencil under people, you can then hover over the faces to show which is which in the picture. Click the pencil on the faceyou want to change / add, it will bring up select face and list the people dectected. If they are not listed (my son was not same situation as you described) you can click the plus symbol to create a new person.
